packages/commons/src/fhir/patient.ts

```diff
@@ -41,7 +41,7 @@ export type Patient = WithStrictExtensions<
 
 export type OpenCRVSPatientName = Omit<fhir3.HumanName, 'use' | 'family'> & {
   use: string
-  family: string[]
+  family?: string[]
```

Comment:
hm not sure probably we had used it like that from the beginning when the typings weren't that strict

Reply:
Highly likely. Could you research what it would take (or if its even possible) for us to get to the standard? One other way I see working here is we keep the array required but make it so that if family name is not submitted, then it is stored as an empty array

Reply:
Okay I'll see if we are actually using it for doing anything useful. Most likely we'll need a migration to get rid of it tho

Reply:
That's alright 👍